test/lint.js

```diff
@@ -132,14 +132,17 @@ describe('linting whole program', function () {
     it('name format', async function () {
       await assertLint(
         positioned`
-          <emu-clause id="foo">
-            <h1>${M}something: ( )</h1>
+          <emu-clause id="a">
```

Comment:
Rather than modifying this test, this should modify the [legal names](https://github.com/tc39/ecmarkup/blob/da698c3b4418e33ed3bd1afdbcac8fd87a0f0bed/test/lint.js#L188) test below.

Reply:
Thanks, I missed that one. Updated.